java/java-hello-world/skaffold.yaml

```diff
@@ -1,15 +1,15 @@
-apiVersion: skaffold/v1beta2
+apiVersion: skaffold/v1beta15
 kind: Config
 build:
   tagPolicy:
     sha256: {}
   # defines where to find the code at build time and where to push the resulting image
   artifacts:
     - image: java-hello-world
-      context: .
       # To learn more about how Jib builds Java containers visit
       # https://github.com/GoogleContainerTools/jib
-      jibMaven: {}
+      jib:
+        project: .
```

Comment:
This `project: .` isn't actually necessary — just use `jib: {}`.  (The `context: .` previously used is also unnecessary — and we should remove it from the dockerfiles profile at the end.

Reply:
Awesome, thanks for the tips. Updating.

Reply:
I also updated it in a different PR that will likely be merged ahead of this, so please sync.